syntax = "proto3";
package bootstrap;

service Bootstrap {
    rpc RequestBootstrap (JoinRequest) returns (JoinResponse) {}
}

message JoinRequest {
    string ip = 1;
    int32 port = 2;
}

message JoinResponse {
    int32 subnetId = 1;

    message Node {
        string ip = 1;
        int32 port = 2;
    }

    repeated Node subnet = 2;
}